spec/features/visitor/visitor_can_add_items_to_cart_spec.rb

```diff
@@ -17,7 +17,7 @@
         expect(current_path).to eq('/cart')
         expect(page).to have_xpath("//img[@src='#{item.image}']")
         expect(page).to have_content(item.title)
-        save_and_open_page
+        # save_and_open_page
```

Comment:
need to remove this

Reply:
Randy's right. Leaving stuff like this in a commit it totally Busch League. Not saying I wouldn't have done it, but that doesn't mean it's ok.